### ISSUE TITLE
Screen Capture by Specific OS

### ISSUE THREAD

**Issue by taufik-nurrohman** [simonw/shot-scraper-template#2]
Does GitHub allow me to create virtual web screen shot via specific operating system? I don&rsquo;t have a Mac but really love the font rendering system. From the workflow file looks like you are using Ubuntu as a virtual operating system used by the actions?

**Comment by simonw** [comment]
Yes it does! I've not tried it myself, but it may be as easy as swapping this line for `macos-latest` instead: https://github.com/simonw/shot-scraper-template/blob/ef42fab14271872f4c280ef316f31a08bb54dfe5/.github/workflows/shots.yml#L9
